Song_sort_Title.css

```diff
@@ -67,62 +41,168 @@ li{
 }
 
 
+
 nav a{
 	display:block;
 	width:100px;
 	padding: 20px 15px;
 	text-decoration:none;
-	font-family:Arial;
-	color:black;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:bold;
+	color:white;
 	text-align: center;
 }
 
 nav a:hover{
-	background:purple;
+	background:black;
 	transition:0.5s;
 }
 
-.content_1{
-	background-color: white;
+ul li ul{
+	padding: 0;
+	position: absolute;
+	top: 58px;
+	width: 100px;
+	display: none;
+	visibility: hidden;
+	z-index: 5;
+}
+
+ul li #a{
+	padding: 0;
+	position: absolute;
+	top: 58px;
+	width: 100px;
+	display: none;
+	visibility: hidden;
+	z-index: 5;
+}
+
+	
+ul li ul li{
+	background-color:  #0C59CF;
+	display: block;
+}
+
+ul li ul li:hover{background: black;}
+
+ul li:hover ul{
+	display: block;
+	visibility: visible;
+}
+
+ul li:hover #a{
+	display: block;
+	visibility: visible;
+}
+
+.logo img{
+	position:absolute;
+	margin-top:-20px;
+}
+
+
+
+.img_1{
 	grid-column-start: 1;
 	grid-column-end: span 4;
-	z-index: 2;	
-	height: 100px;
-	border: 2px solid red;
+}
+
+.img_1 img{
+	display:block;
+	margin-left:auto;
+	margin-right:auto;
+	height:400px;
+	width:100%;
+}
+
+.footer{
+	background-color: blue;
+	grid-column-end: span 4;
+	padding-top:25px;
+	text-align: center;
+	color:white;
+	height:50px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
+}
+
+
+.content_1{
+	grid-column-start: 2;
+	grid-column-end: span 4;
+	background-color: white;
+	z-index: 2;
+	height:200px;
+	text-align:center;
+}
+
+.section3{
+	background-color: white;
+	width: 85%;
+	height: 200px;
+	position: absolute;
+	z-index: 4;
 }
 
 
+
+
+.section3 h1{
+	font-size:24px;
+	color:black;
+	margin-top:50px;
+	font-family: 'Source Sans Pro', sans-serif;
+	font-weight:600;
+}
+
 .content_2{
 	grid-column-start: 1;
-	grid-column-end: span 1;
+	grid-column-end: 2;
```

Comment:
21/9: I set the column end to 2 because I want to this div the one that hold the img that I decided to make this page more aesthetic to attract people attention.(15% of the page)